packages/protocol/thread/pause.ts

```diff
@@ -190,6 +190,10 @@ export class Pause {
     datas.forEach(d => this._updateDataFronts(d));
   }
 
+  isLoaded() {
```

Comment:
Can we change the name of this method? `isLoaded` suggests it checks if the pause is loaded and returns a boolean, so someone is bound to do `if (pause.isLoaded) {...}` eventually.